exotica_core/include/exotica_core/dynamics_solver.h

```diff
@@ -130,14 +130,15 @@ class AbstractDynamicsSolver : public Object, Uncopyable, public virtual Instant
     void SetIntegrator(std::string integrator_in);
 
     /// \brief Returns the control limits vector.
-    Eigen::VectorXd get_control_limits();
-    void set_control_limits(Eigen::VectorXd control_limits);
+    Eigen::VectorXd get_control_limits_low();
```

Comment:
For get we usually return a matrix where the first column is the lower and the second column is the upper limit. Could you please change it to that as well? It matches the kinematic limits etc.